x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/integration_tests/response_actions_log.test.tsx

```diff
@@ -1544,9 +1545,9 @@ describe('Response actions history', () => {
       await user.click(getByTestId(`${testPrefix}-${filterPrefix}-popoverButton`));
       const filterList = getByTestId(`${testPrefix}-${filterPrefix}-popoverList`);
       expect(filterList).toBeTruthy();
-      expect(getAllByTestId(`${filterPrefix}-option`).length).toEqual(
-        RESPONSE_ACTION_API_COMMANDS_NAMES.length
-      );
+      // TODO: RUNSCRIPT does not have a corresponding action in the API yet
+      const TEMPORARY_LENGTH = RESPONSE_ACTION_API_COMMANDS_NAMES.length - 1;
+      expect(getAllByTestId(`${filterPrefix}-option`).length).toEqual(TEMPORARY_LENGTH);
```

Comment:
This is an issue with the EUI popover component. It shows 9 items at a time on the default height. We need to collect the options using `querySelectorAll` or something like that. I think worth noting this glitch in a comment here 😓

Reply:
Ok @ashokaditya showed me on zoom, that this was indeed the case. I was adjusting wrong components height 😅 Thanks!

Reply:
I found a workaround and sent you the diff.

Reply:
Applied, thanks a lot 👍